app.js

```diff
@@ -7,6 +7,13 @@ const govukTemplate = require('hof-govuk-template');
 const hoganExpressStrict = require('hogan-express-strict');
 const expressPartialTemplates = require('express-partial-templates');
 const _ = require('lodash');
+const apiCall = require('./lib/api-call');
+const config = require('./config');
+const baseUrl = config.apiBaseUrl;
```

Comment:
config.var :thinking: sorry basic question time, I've googled config.var and err I didn't quite find anything except referring to variables in my config so didn't quite understand this 

object deconstructing? ahh just read up about it, I did not know that quality, you mean this right? http://wesbos.com/destructuring-objects/